docs/content/en/3.providers.md

```diff
@@ -1,20 +1,20 @@
 ---
 title: Providers
-description: 'Nuxt Image supports multiple providers for high performances.'
-category: 'Getting Started'
+description: "Nuxt Image supports multiple providers for high performances."
+category: "Getting Started"
 ---
 
-## Static Images
+## Default provider
 
-For static projects (`target: 'static'`) image module automatically optimizes `src` for both local and external assets via a preconfigured instance of [IPX](/providers/ipx) during development and generation (`nuxt generate`) so you don't have to worry about choosing a hosting.
+Nuxt Image comes with a preconfigured instance of [IPX](/providers/ipx) to provide images transformations based on the [Sharp](https://www.npmjs.com/package/sharp) package. That default provider can be used without any configuration. The module will automatically optimize `<nuxt-img>` or `<nuxt-picture>` sources and accepts all [options](/api/options/), except for modifiers which are specific to image services providers.
 
-## Image Providers
+With Universal target, IPX will generate correct images at runtime. With static target, IPX will generate all optimized assets on `nuxt generate` and store them in the `dist` folder.
 
-With a server/serverless target or with dynamic images that aren't known at generation-time we need a runtime image provider.
+## Image Services Providers
```

Comment:
Providers are not necessarily a service. Like vercel is integrated cdn. And ipx is replaceable with a custom non-service provider